.gitignore

```diff
@@ -17,8 +17,11 @@ crawlers/ecolex/literature/*
 !crawlers/ecolex/literature/.*
 
 crawlers/eurlex/files/*
-!crawlers/eurlex/files/.*
+crawlers/eurlex/!files/.*
 crawlers/eurlex/celex_nums/*
 
+crawlers/eurlex/celex_nums/*
+crawlers/eurlex!celex_nums/.*
```

Comment:
Should this be `crawlers/eurlex/!celex_nums/.*`?

Also duplicated `crawlers/eurlex/celex_nums/*`